CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ option(Autoscoper_BUILD_WITH_CUDA "Build With CUDA instead of OpenCL" ON)
 #-----------------------------------------------------------------------------
 # Superbuild Option - Enabled by default
 #-----------------------------------------------------------------------------
+include(Cmake/ExternalProjectDependency.cmake)
```

Comment:
Move this after setting `SUPERBUILD_TOPLEVEL_PROJECT`. This will ensure the value we set is used. See https://cmake-artichoke.readthedocs.io/en/latest/ExternalProjectDependency.html#variable:SUPERBUILD_TOPLEVEL_PROJECT